packages/web-components/@groupby/elements-search-box/stories/index.ts

```diff
@@ -11,6 +17,16 @@ const updateTextEvent = [
   },
 ];
 
+const eventsToListen = [
+  SEARCHBOX_CLEAR,
+  SEARCHBOX_CLICK,
+  SEARCHBOX_INPUT,
+  SEARCH_REQUEST,
+  UPDATE_SEARCH_TERM
+];
+
+addStorybookListeners(eventsToListen);
```

Comment:
@ssbelle Johann's comment about inlining this still applies here. If you're not going to use `eventsToListen` again, you can simply do:
```suggestion
addStorybookListeners([
  SEARCHBOX_CLEAR,
  SEARCHBOX_CLICK,
  SEARCHBOX_INPUT,
  SEARCH_REQUEST,
  UPDATE_SEARCH_TERM,
]);
```